### PR TITLE
Updates blogging prompts to use v3 endpoint

### PR DESCRIPTION
**Pull request by creativecoder**
## Proposed Changes

Updates blogging prompt query hook to use wpcom/v3 endpoint

Important: should be merged when mobile apps are updated. See pctCYC-KA-p2#comment-862

## Testing Instructions

* Make sure your site is set up to display posts on the home page
* Visit My Home and see that blogging prompts display correctly in the blogging prompts card\

image

### CODE REVIEW

File: client/data/blogging-prompt/use-blogging-prompts.ts [hunk 1]
```diff
@@ -1,42 +1,40 @@
...
+	const today = moment().format( '--MM-DD' );
 
 	return useQuery( {
-		queryKey: [ 'blogging-prompts', today + '-' + page ],
+		queryKey: [ 'blogging-prompts', siteId, today, per_page ],
 		queryFn: () =>
 			wp.req.get( {
-				path: `/sites/${ siteId }/blogging-prompts?number=${ page }&from=${ today }`,
-				apiNamespace: 'wpcom/v2',
+				path: `/sites/${ siteId }/blogging-prompts?per_page=${ per_page }&after=${ today }&order=desc`,
```
Comment (flootr) [inline client/data/blogging-prompt/use-blogging-prompts.ts]: 👋🏻 Howdy! Just a drive-by comment that you can pass in a `query` object as second parameter to `wp.req.get` and it'll automatically append that to your path. That way you don't have to construct the string manually and it's a little more comfortable to add and remove query params. Like this:

```js
wp.req.get( {
  path: `/sites/${ siteId }/blogging-prompts`,
  apiNamespace: 'wpcom/v3'
}, {
  per_page,
  after: today,
  order: 'desc'
} )
```

**Comment by roo2** [comment]
rebased.

**Comment by roo2** [comment]
oops, merged PRs in the wrong order 🤦 this now contains https://github.com/Automattic/wp-calypso/pull/83696. But we have the go ahead to merge this anyway and I've retested so I'll merge these now. 🚢

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/9619521

Thank you @creativecoder for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/9619521).
